Src/FluentAssertions/Equivalency/XDocumentEquivalencyStep.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Xml.Linq;
+
+namespace FluentAssertions.Equivalency
+{
+    public class XDocumentEquivalencyStep : EquivalencyStep<XDocument>
+    {
+        protected override EquivalencyResult OnHandle(Comparands comparands, IEquivalencyValidationContext context, IEquivalencyValidator nestedValidator)
+        {
+            var subject = (XDocument)comparands.Subject;
+            var expectation = (XDocument)comparands.Expectation;
+
+            subject.Should().BeEquivalentTo(expectation, context.Reason.FormattedMessage, context.Reason.Arguments);
```

Comment:
I haven't thought through if this is problematic, but I observe that this does not take an `EquivalencyAssertionOptions`.
So it cannot be customized unless setting the overrides globally.

Reply:
There's nothing to customize right now. See https://github.com/fluentassertions/fluentassertions/blob/develop/Src/FluentAssertions/Xml/XDocumentAssertions.cs#L83